lgc/state/PipelineState.cpp

```diff
@@ -540,6 +540,20 @@ void PipelineState::readUserDataNodes(Module *module) {
   m_userDataNodes = ArrayRef<ResourceNode>(m_allocUserDataNodes.get(), nextOuterNode);
 }
 
+// =====================================================================================================================
+// Find the push constant resource node.
+// Returns {topNode, node} where "node" is the found user data node, and "topNode" is the top-level user data
```

Comment:
Please fix the comment as it only returns a top-level node.

Reply:
Done.